linkml-schema/kbmodel.yaml

```diff
@@ -96,34 +78,222 @@ classes:
+  barcoded cell sample:
+    is_a: process output
+    attributes:
+      port well:
+      sample quallity count:
+
+
+  # 022 - cnda_amplification.ipynb
+  amplified cdna:
+    is_a: process output
+    attributes:
+      amplified quantity:
+        description: >-
+          Amount of cDNA amplified (ng).
+        range: float
+      pcr cycles:
+      percent cdna longer than 400bp:
+      rna amplification pass fail:
+    slots:
+      - method
+
+
+  # 021 - library_constructions.ipynb
+  library:
+    is_a: process output
+    attributes:
+      avg size:
+        description: The average size (bp) of the library fragments. bp stands for base pairs.
+        range: integer
+      input amount:
+        description: The amount of input material (ng) used to create the library.
+        range: float
+      library prep pass:
+      quantification fmol:
+        range: float
+      quantification ng:
+        range: float
+      quantification nm: 
+        range: float
+      r1 index:
+      r1 sequence:
+      r2 index:
+      r2 sequence:
+    slots:
+      - method 
+      - creation date
+    slot_usage:
+      creation date:    
+        description: The date and time the object was created. 
+        range: datetime
+      
+
+  # 020 - aliquoting_and_pooling.ipynb
+  library aliquot:
+    is_a: process output
+    attributes:
+      input quantity:
+        description: The amount of input material (fmol) used to create the library aliquot.
+        range: integer
+
+
+  # 020 - aliquoting_and_pooling.ipynb
+  library pool:
+    is_a: process output
+    attributes:
+      tube avg size:
+      tube contents:
+        description: The content concentration (nm).
+        range: float
+      tube internal label:
+
+
 slots:
   referenced in:
     range: genome annotation
+
   molecular type:
     range: BioType
+
   reference assembly:
     range: uriorcurie
+
   digest:
+    description: Stores checksum information. 
     range: checksum
     multivalued: true
     required: true
+
   content_url:
     slot_uri: schema:url
     multivalued: true
-  gene_identifier_1:
-    range: gene annotation
-  gene_identifier_2:
-    range: gene annotation
+
   authority:
-  method:
-  creation_date:
-  access_date:
+    description: The organization responsible for publishing the data. 
+
   checksum algorithm:
+    description: The type of cryptographic hash function used to calculate the checksum value.
     range: DigestType
-  value:
+
   source id:
     description: The authority specific identifier. 
     slot_uri: schema:identifier
+  
+  strain:
+    description: The genetic variant or subtype of a species or organism. 
+
+  label:
+    description: The name of the object.
+    slot_uri: schema:name
+
+  sex:
+    range: SexType  
+  
+  date of birth:
+    description: The date of birth of the donor.
+    range: date
+
+  date of death:
+    description: The date of death of the donor.
+    range: date
+  
+  age at death:
```

Comment:
I think they didn't use int but time duration from ISO 8601

Reply:
`range: iso8601duration` was not working. I instead used `range: string`. Using this example I found: https://genomicsstandardsconsortium.github.io/mixs/0001195/